templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -66,6 +66,7 @@ patterns like `#1234` to your ticketing system.
 Auto-detected URL: zulipchat.com
 Named link: [Zulip homepage](zulipchat.com)
 Stream: #**announce**
+Stream and Topic: #**general>a topic name**
```

Comment:
We also need to regenerate `/static/images/help/markdown-links.png` (I'd do it after changing the `announce` in the line above to `general` as well.

Note that to generate the screenshot you'll need to change to `line-height: 22px` and `font-size: 16px` in `.message-content`.

Reply:
Maybe this should be done in a followup commit.